src/ToCore/Type.ml

```diff
@@ -51,11 +49,9 @@ let rec tr_effrow_end env (eff : S.Type.effrow_end) =
 and tr_type env tp =
   match S.Type.view tp with
   | TUnit    -> T.Type.Ex TUnit
-  | TUVar _  ->
-    (* TODO: they can be supported, and its especially useful in REPL *)
-    InterpLib.Error.report ~cls:FatalError
-      "Unsolved unification variables left.";
-    raise InterpLib.Error.Fatal_error
+  | TUVar (_,uvar)  ->
+    let uid = Lang.Unif.UVar.uid uvar in
+    T.Type.Ex (TUVar (uid, KType))
```

Comment:
Some comment as above applies here. Moreover, the kind of this variable is not necessarily `KType`. It should be a translated kind of this variable (it could be an unification variable of e.g. arrow kind).